### PR TITLE
VIH-8099 added moj style module for header component

### PR DESCRIPTION
**Pull request by omio-ma**
### JIRA link ###
https://tools.hmcts.net/jira/browse/VIH-8099

### Change description ###
- added moj node module
- updated header component to use moj styles

**Does this PR introduce a breaking change?** 

```
[  ] Yes
[ x ] No
```

### CODE REVIEW

**Comment by FarronF** [comment]
Why the change from govuk to moj?

**Comment by omio-ma** [comment]
> Why the change from govuk to moj?

@FarronF thats been suggested by Neil or are we not meant to be using i?

File: VideoWeb/VideoWeb/ClientApp/src/app/shared/header/header.component.html [hunk 1]
```diff
@@ -1,33 +1,35 @@
...
...0 .1.2.2.3.2zM22 7c.2 0 .3-.1.3-.3 0-.2-.1-.3-.3-.3s-.3.1-.3.3c0 .2.1.3.3.3zm2.2.8c.2 0 .3-.1.3-.3 0-.2-.1-.3-.3-.3s-.3.1-.3.3c0 .2.2.3.3.3zm-.5-.6c.2 0 .3-.1.3-.3 0-.2-.1-.3-.3-.3s-.3.1-.3.3c-.1.2 0 .3.3.3zm-.9-.2c.2 0 .3-.1.3-.3 0-.2-.1-.3-.3-.3s-.3.1-.3.3c-.1.1.2.3.3.3zm1.8 2.4c-.2 0-.3.1-.3.3 0 .2.1.3.3.3s.3-.1.3-.3c0-.2-.1-.3-.3-.3zm.1-.7c-.2 0-.3.1-.3.3 0 .2.1.3.3.3s.3-.1.3-.3c0-.2-.1-.3-.3-.3zm-.4-.5c0 .2.1.3.3.3s.4-.1.4-.3c0-.2-.1-.3-.3-.3s-.4.1-.4.3zm-5.9-.5c.2 0 .3-.1.3-.3 0-.2-.1-.3-.3-.3s-.3.1-.3.3c0 0 .2.3.3.3zm-.8-.5c.2 0 .3-.1.3-.3 0-.2-.1-.3-.3-.3s-.3.1-.3.3c.1.2.2.3.3.3zm-.7-.2c.2 0 .3-.1.3-.3 0-.2-.1-.3-.3-.3s-.3.1-.3.3c-.1.2.1.3.3.3zm-2.3.8c.2 0 .3-.1.3-.3 0-.2-.1-.3-.3-.3s-.3.1-.3.3c0 .2.1.3.3.3zm.6-.6c.2 0 .3-.1.3-.3 0-.2-.1-.3-.3-.3s-.3.1-.3.3c-.1.2.2.3.3.3zm.9-.2c.2 0 .3-.1.3-.3 0-.2-.1-.3-.3-.3s-.3.1-.3.3c-.1.1.1.3.3.3zm-1.5 2.8c0-.2-.1-.3-.3-.3s-.3 0-.3.3c0 .2.1.3.3.3.1 0 .3-.1.3-.3zm-.2-.8c0-.2-.1-.3-.3-.3-.2 0-.3.1-.3.3 0 .2.1.3.3.3.2 0 .3-.2.3-.3zm-.2-.5c.2 0 .3-.1.3-.3 0-.2-.1-.3-.3-.3s-.4.1-.4.3c0 .1.3.3.4.3zm7.7.3c-.1.1-.4.4-.4.8 0 .2.2.6.3.8.1-.3.4-.4.4-.8 0-.3-.2-.5-.3-.8zm-4.9 0c-.2.3-.3.6-.3.9 0 .3.3.4.4.8.1-.3.3-.6.3-.8 0-.5-.3-.8-.4-.9z"></path>
```
Comment (shaed-parkar) [inline VideoWeb/VideoWeb/ClientApp/src/app/shared/header/header.component.html:8]: any chance the commands can be moved out into a class?

Comment (omio-ma) [reply VideoWeb/VideoWeb/ClientApp/src/app/shared/header/header.component.html]: moved it into a separate template file, thanks for the suggestion, I left it in the shared folder

File: VideoWeb/VideoWeb/ClientApp/src/app/shared/header-logo-svg/header-logo-svg.component.html [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<svg role="presentation" focusable="false" class="moj-header__logotype-crest" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40" height="40" width="40">
```
Comment (FarronF) [inline VideoWeb/VideoWeb/ClientApp/src/app/shared/header-logo-svg/header-logo-svg.component.html:1]: https://css-tricks.com/accessible-svgs/
https://stackoverflow.com/questions/4697100/accessibility-recommended-alt-text-convention-for-svg-and-mathml

Add role="img" and aria-label="somethingHere" to the svg
Add a <title> and <description> inside as first children.

Make sure there is welsh translations for these.

File: VideoWeb/VideoWeb/ClientApp/src/app/shared/header-logo-svg/header-logo-svg.component.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  selector: 'app-header-logo-svg',
+  templateUrl: './header-logo-svg.component.html',
+  styleUrls: ['./header-logo-svg.component.css']
+})
+export class HeaderLogoSvgComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void {
+  }
```
Comment (FarronF) [inline VideoWeb/VideoWeb/ClientApp/src/app/shared/header-logo-svg/header-logo-svg.component.ts:13]: Remove empty constructor and function.

**Comment by FarronF** [comment]
> 
> 
> > Why the change from govuk to moj?
> 
> @FarronF thats been suggested by Neil or are we not meant to be using this module?

That's totally fine. Just wondering if it was discussed with Neil as it wasn't on the ticket.

**Review by FarronF — APPROVED** [review]

**Comment by FarronF** [comment]
Sorry didn't see this before but remove empty css file since that gets marked as a major code smell by sonar cloud.

**Review by FarronF — APPROVED** [review]

**Review by kevin-hmcts — APPROVED** [review]
LGTM

**Pull request merged**